test/versioned/elastic/package.json

```diff
@@ -22,7 +22,8 @@
         "node": ">=16"
       },
       "dependencies": {
-        "@elastic/elasticsearch": ">=7.16.0 <8.5.0"
+        "@elastic/elasticsearch": ">=7.16.0",
```

Comment:
this is going to run the same tests on 18 and 20 as the stanza below.  I would restore the version ranges from before